jq/script-01.sh

```diff
@@ -5,3 +5,5 @@ set -euo pipefail
 # The input for this script is the person.json file.
 # TODO: Write a command to output the name of the person.
 # Your output should be exactly the string "Selma", but should not contain any quote characters.
+jq -r .'name' person.json
+Selma
```

Comment:
It is useful to see the output produced, but we'd generally avoid putting this directly in a shell script, as it causes the script to no longer run, e.g.

```console
% ./script-01.sh
Selma
./script-01.sh: line 9: Selma: command not found
```

If you're going to include the output, I'd recommend doing so in a comment (i.e. putting a `#` before the output) so that the scripts can still run :)